### PR TITLE
Add instructions to externalize Grafana service.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: docs/influxdb.md [hunk 1]
```diff
@@ -12,6 +12,23 @@ $ kubectl.sh create -f deploy/kube-config/influxdb/
 
 Grafana will be accessible at `https://<masterIP>/api/v1beta1/proxy/services/monitoring-grafana/`. Use the master auth to access Grafana.
 
+## Production setup for Influxdb and Grafana
```
Comment (vmarmol) [inline docs/influxdb.md]: nit: InfluxDB

File: docs/influxdb.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ $ kubectl.sh create -f deploy/kube-config/influxdb/
...
+
+By default, InfluxDB and Grafana are made accessible via a proxy on Kubernetes apiserver, to make them accessible with minimal configurations on all
```
Comment (vmarmol) [inline docs/influxdb.md]: on "the" Kubernetes apiserver

File: docs/influxdb.md [hunk 3]
```diff
@@ -12,6 +12,23 @@ $ kubectl.sh create -f deploy/kube-config/influxdb/
...
+Kubernetes deployments.
+This is not ideal for production deployments since the proxy will be a bottleneck. 
+To use Influxdb and Grafana in production, we recommend setting up an [External Kubernetes Service](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services).
```
Comment (vmarmol) [inline docs/influxdb.md]: nit: InfluxDB we're not consistent :)

File: docs/influxdb.md [hunk 4]
```diff
@@ -12,6 +12,23 @@ $ kubectl.sh create -f deploy/kube-config/influxdb/
...
+
+#### Guide
+1. Remove the environment variable `INFLUXDB_EXTERNAL_URL` from InfluxDB Grafana RC config [here](../deploy/kube-config/influxdb/influxdb-grafana-controller.json).
+   This option makes Grafana reach InfluxDB service via the apiserver proxy.
+
+2. Update Grafana service to either include a public IP or setup an external load balancer.
+   In the [Grafana service](../deploy/kube-config/influxdb/grafana-service.json),  set `"createExternalLoadBalancer": true` in the Grafana Service Spec or,
+   set `"PublicIPs": ["<Externally Accessible IP>"]`.
+
+3. Delete and recreate InfluxDB and Grafana Rcs and services.
```
Comment (vmarmol) [inline docs/influxdb.md]: nit: RCs

**Comment by vmarmol** [comment]
Just nits

**Comment by vishh** [comment]
Fixed all nits. Thanks for the review @vmarmol

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**